src/Layout/Navigation/__tests__/RecentList.spec.tsx

```diff
@@ -29,45 +29,49 @@ jest.mock('react-tooltip', () => {
   };
 });
 
+const cheWorkspaces: che.Workspace[] = [
```

Comment:
Do you mind using createFakeCheWorkspace here?

Reply:
https://github.com/eclipse/che-dashboard/blob/master/src/store/__mocks__/workspace.ts#L15

Reply:
const workspaces = [0, 1, 2, 3, 4].map(i => createFakeCheWorkspace('wksp-' + i, 'wksp-' + i));

Reply:
that makes sense, thanks